tst/EnergyPlus/unit/HighTempRadiantSystem.unit.cc

```diff
@@ -137,5 +141,35 @@ namespace EnergyPlus {
 
 	}
 
+	TEST_F( EnergyPlusFixture, HighTempRadiantSystemTest_SizeHighTempRadiantSystemScalableFlagSetTest )
+	{
+		int RadSysNum;
+		int SizingTypesNum;
+		
+		HighTempRadiantSystem::clear_state();
```

Comment:
This doesn't need to be here.  I'll remove it.  It gets called from the fixture.